Documentation/ApiOverview/FormEngine/DataCompiling/Index.rst

```diff
@@ -172,42 +172,30 @@ it, :php:`TcaFlexProcess` uses the prepared data structure, processes values and
 from the extension "news" hooks in between these two to add some own preparation stuff. The registration happens with this
 code in :file:`ext_localconf.php`:
 
-.. code-block:: php
-
-    // Modify flexform fields since Core 8.5 via formEngine: Inject a data provider
-    // between TcaFlexPrepare and TcaFlexProcess
-    if (\TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 8005000) {
-        $GLOBALS['TYPO3_CONF_VARS']['SYS']['formEngine']['formDataGroup']['tcaDatabaseRecord']
-        [\GeorgRinger\News\Backend\FormDataProvider\NewsFlexFormManipulation::class] = [
-            'depends' => [
-                \TYPO3\CMS\Backend\Form\FormDataProvider\TcaFlexPrepare::class,
-            ],
-            'before' => [
-                \TYPO3\CMS\Backend\Form\FormDataProvider\TcaFlexProcess::class,
-            ],
-        ];
-    }
+..  literalinclude:: _ext_localconf.php
```

Comment:
Currently, this snippet is not used anymore in EXT:news. We can leave it, but add a hint that it is a former version of news we are referring to. What do you think?

https://github.com/georgringer/news/blob/main/ext_localconf.php#L104-L112

Reply:
Added a comment